meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -2,12 +2,19 @@
 
 source /usr/sbin/resin-vars
 
+# The tmp supervisor.conf is used by the supervisor updater
+if [ -f /tmp/update-supervisor.conf ]; then
+	source /tmp/update-supervisor.conf
+else
+	source /etc/resin-supervisor/supervisor.conf
+fi
+
 SUPERVISOR_IMAGE_ID=$(balena inspect --format='{{.Id}}' $SUPERVISOR_IMAGE:$SUPERVISOR_TAG)
 SUPERVISOR_CONTAINER_IMAGE_ID=$(balena inspect --format='{{.Image}}' resin_supervisor || echo "")
 
 runSupervisor() {
     balena rm --force resin_supervisor || true
-    balena run --privileged --name resin_supervisor \
+    balena run -d --privileged --name resin_supervisor \
```

Comment:
For balena to manage the container, don't we need to set a restart policy? i.e. `--restart=always`?

Reply:
I was about to do it in another PR but I think that given it fixes restarts this PR should include this too. I'll add it.